examples/text-fields-attribute.rs

```diff
@@ -78,7 +78,7 @@ impl Widget for Win {
                 #[name="entry"]
                 gtk::Entry {
                     changed(entry) => {
-                        let text = entry.get_text().expect("get_text failed");
+                        let text = entry.get_text().expect("get_text failed").as_str().to_owned();
```

Comment:
Instead of converting it to `String`, please store a `GString` in the `Change` message.

Reply:
I store GString in some places, some not. GString doesn't have Clone so it is not usable in where Clone is required. In some places it is more convenient to use String.